todoitem.css

```diff
@@ -0,0 +1,163 @@
+/** 
+  * todoitem.css
+  *
+  * These are the styles for the To-do Items on the webpage    
+  *
+  * Index
+  * - badge
+  * - badge-danger
+  * - badge-dark
+  * - badge-info
+  * - badge-light
+  * - badge-pill
+  * - badge-secondary
+  * - badge-success
+  * - badge-warning
+  * - btn (properties shared with .badge)
+  * - btn-del-single
+  * - btn-edit
+  * - my-4
+  * - name
+  * - #option
+  * - tag
+  * - task-option
+  * - to-do-item
+  */
+
+.badge {
+    border-radius: .25rem;
+    display: inline-block;
+    font-size: 75%;
+    font-weight: 700;
+    line-height: 1;
+    padding: .25em .4em;
+    text-align: center;
+    vertical-align: baseline;
+    white-space: nowrap;
+}
+.badge .btn {
+    position: relative;
+    top: -1px;
+}
+.badge-danger {
+    background-color: #dc3545;
+    color: #fff;
+} 
+.badge-danger[href]:focus, .badge-danger[href]:hover {
+    background-color: #bd2130;
+    color: #fff;
+    text-decoration: none;
+}
+.badge-dark {
+    background-color: #343a40;
+    color: #fff;
+}
+.badge-dark[href]:focus, .badge-dark[href]:hover {
```

Comment:
yea imo anything with more than one class/id, it gets hard to read without breaking it with newlines

Reply:
@mentosfreshmaker, I put each class into a new row in the cases where multiple properties are shared.